lib/alertify.js

```
@@ -356,10 +361,11 @@
 		 * 
 		 * @param  {String} message    The message passed from the callee
 		 * @param  {String} type       [Optional] Optional type of log message
+		 * @param  {Number} wait       [Optional] Time (in ms) to wait before auto-hiding the log
 		 * 
 		 * @return {Object}
 		 */
-		log = function (message, type) {
+		log = function (message, type, wait) {
```

maybe throw an error catching statement here

``` js
if (wait && typeof wait !== "number") throw new Error("wait parameter must be a number");
```